ZCRMiOS/Classes/ZCRMDashBoard.swift

```diff
@@ -0,0 +1,123 @@
+//
+//  ZCRMDashBoard.swift
+//
+//  Created by Kalyani shiva on 12/07/18.
+//
+
+import Foundation
+
+public class ZCRMDashBoard{
+    
+    fileprivate var dashBoardID = Int64()
+    fileprivate var dashBoardName = String()
+    fileprivate var isSystemGenerated = Bool()
+    fileprivate var isSalesTrends = Bool()
+    fileprivate var accessType = String()
+    fileprivate var dashBoardMetaComponent:[ZCRMDashBoardMetaComponent]?
+    
+    struct Properties {
+        
+        static let jsonRootKey = "Analytics"
+        
+        struct URLPathName {
+            static let ANALYTICS = "Analytics"
+            static let COMPONENTS = "components"
+            static let REFRESH = "refresh"
+            static let COLORTHEMES = "color_themes"
+        }
+        
+        struct APINames {
```

Comment:
APINames -> Param keys

Reply:
Can you please elaborate? I didn't understand .

Reply:
These are param keys, not API names.